src/Symfony/Component/Form/Attribute/Type.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Attribute;
+
+/**
+ * Add an AsFormType class property as a FormType's field.
+ *
+ * @author Benjamin Georgeault <git@wedgesama.fr>
+ */
+#[\Attribute(\Attribute::TARGET_PROPERTY)]
+final readonly class Type
+{
+    /**
+     * @param class-string|null    $type
+     * @param array<string, mixed> $options
+     */
+    public function __construct(
+        private ?string $type = null,
+        private array $options = [],
```

Comment:
What about also adding `?string $name = null`, which defaults to the property name if null? If it’s set, it would use property_path for mapping. This could be convenient in cases where you need to change the field name in the view